commons/libs/constants/ApiResponseMessages.ts

```diff
@@ -0,0 +1,4 @@
+export const CREATE_DONATION_REQUEST_SUCCESS = 'We have accepted your request, and we will let you know when we find a donor.'
+export const UPDATE_DONATION_REQUEST_SUCCESS = 'We have updated your request and will let you know once there is an update.'
+export const CREATE_PROFILE_SUCCESS = 'Your Profile info has been added.'
```

Comment:
your profile has been created?

Reply:
Updated in f98450b71743d5473aa397d4854e7347002cc21c